geode-core/src/main/java/org/apache/geode/internal/SystemAdmin.java

```diff
@@ -1909,10 +1900,7 @@ private static boolean needsSysDir(String cmd) {
     if (cmd.equalsIgnoreCase("version")) {
       return false;
     }
-    if (cmd.equalsIgnoreCase("help")) {
-      return false;
-    }
-    return true;
+    return !cmd.equalsIgnoreCase("help");
```

Comment:
If you want to collapse this block, I think this might read a little better.

```
if (cmd.equalsIgnoreCase("stats")
    || cmd.equalsIgnoreCase("merge-logs")
    || cmd.equalsIgnoreCase("version")
    || cmd.equalsIgnoreCase("help")) {
    return false;
}
return true;
```

I guess you could collapse that to a one-liner, but I don't think that would read as well.